integration_tests/src/main/python/benchmark.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+
+def main():
+    """Iterate over a series of configurations and run benchmarks for each of the specified
+    queries using that configuration.
+
+    Example usage:
+
+    python benchmark.py
+      --benchmark tpcds \
+      --input /path/to/input \
+      --input-format parquet \
+      --output /path/to/output \
+      --output-format parquet \
+      --configs cpu gpu-ucx-on
+      --query q4 q5
+
+    In this example, configuration key-value pairs will be loaded from cpu.properties and
+    gpu-ucx-on.properties and appended to a spark-submit-template.txt to build the spark-submit
```

Comment:
this is amazing. I would however add a sample of what a properties file should look like. I assume it's `key=value` but I guess it could be `key value`.

Reply:
I see we use `=`. Could we use **https://docs.python.org/3.5/library/configparser.html**?

That may even give you the flexibility to have multiple sections in one config as well.

Reply:
That said this could be a follow up.

Reply:
Thanks, that config parser library would help a lot actually because I could distinguish between the different types of config that spark-submit accepts (those that are provided with `--conf` and those that are not, like scheduler-specific settings). I will look at doing this as a follow-up.

Reply:
Filed as https://github.com/NVIDIA/spark-rapids/issues/921